### DISCUSSION TITLE
Custom Asset manager not showing up in full screen mode

### DISCUSSION THREAD

**Discussion by rajmadhawk** [GrapesJS/grapesjs#4081] [Q&A]
Hi 
I have integrated as custom asset manager by the following method 

 editor  = grapesjs.init({
  ............................
              assetManager: {
                  custom: true,
                  }
...................................
});

and calling it 
 editor.on('asset:custom', props => {  
		  props.container.appendChild(open_customEditor('filename'));
		  });

The functionality works as desired when in regular mode. However, when in fullscreen mode, the modal for custom asset manager shows behind the fullscreen and not on top.
Is there any setting that needs to be maintained for the custom asset manager to show in full screen mode.

**Comment by artf** [comment]
can add a demo of your example?
